tools/updateVersion.ps1

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+param(
+    [Parameter(Mandatory)]
+    [semver]$Version,
+
+    [Parameter(Mandatory)]
+    [string]$Changes
+)
+
+git diff --staged --quiet --exit-code
+if ($LASTEXITCODE -ne 0) {
+    throw "There are staged changes in the repository. Please commit or reset them before running this script."
+}
+
+$Path = "Directory.Build.props"
+$f = Get-Content -Path $Path
+$f = $f -replace '^(?<prefix>\s+<ModuleVersion>)(.+)(?<suffix></ModuleVersion>)$', "`${prefix}${Version}`${suffix}"
```

Comment:
```suggestion
$f = $f -replace '^<ModuleVersion>.+</ModuleVersion>', "<ModuleVersion>${Version}</ModuleVersion>"
```

Reply:
Keeping as-is for later copy/paste/modify.